pkg/operator/ceph/cluster/controller.go

```diff
@@ -460,7 +460,7 @@ func (c *ClusterController) configureLocalCephCluster(namespace, name string, cl
 				logger.Errorf(failedMessage)
 				if !canRetry {
 					// it may seem strange to exit true but we don't want to retry if the version is not supported
-					return true, nil
+					return true, err
```

Comment:
So if we return `true, err`, the `wait.Poll()` will exit since it's `true`, then the `err != nil` will be triggered on line 483 to abort the reconcile?

Reply:
correct